integration-tests/api/__tests__/admin/discount.js

```diff
@@ -991,7 +1009,6 @@ describe("/admin/discounts", () => {
     })
 
     it("fails to create a fixed discount with multiple regions", async () => {
-      expect.assertions(2)
```

Comment:
todo: you can't remove it in places where the expects are done in a catch block. either they need to be refactored, but it is out of the scope of that pr, or we have to let it here to prevent the call to succeed and the test to pass because the expects have not been used. I let you revert it if you have done it in other places like this one